microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -103,6 +103,15 @@ You can delete templates stored on the device and clear the cache data. Go to `e
 You can use the title of your collection to find relevant Pinterest Topic pages when searching in Microsoft Bing. Go to `edge://settings/privacy`. Turn on the **Show suggestions from Pinterest in Collections** setting. Microsoft Edge does not send data about your collections to Pinterest. You can remove the suggestions and stop searches for Pinterest Topic pages. Go to `edge://settings/privacy` and turn off the **Show suggestions from Pinterest in Collections** setting.
 Collections aren't available when using **InPrivate** or **Guest** modes.
 
+<!-- ====================================================================== -->
+## Compose
+
+In the Bing side pane, Compose uses AI to write for you starting from a short prompt and "tone", "format", and "length" selections. Text generated by Compose can be copied or inserted into the text page directly. To access Compose, click on the blue Bing icon in the right hand pane > Compose (requires Bing waitlist approval). Text and selections inputed by the user is sent to an internal Microsoft Large Language Model (LLM).
+
+![Compose](./media/compose.png)
+
+The request is sent over a secure HTTPS connection with encryption and contains the input prompt, formatting selections, along with IDs associated with the user's Edge profile and device, to reference for subsequent requests. User input is not stored at all. 
```

Comment:
```suggestion
The request is sent over a secure HTTPS connection and contains the input prompt, formatting selections, along with IDs associated with the user's Edge profile and device, to reference for subsequent requests. User input is not stored at all. 
```

Optional suggestion: "with encryption" could be left out since that's already implied with HTTPS.

Reply:
confirmed eg wikip: "Hypertext Transfer Protocol Secure (HTTPS) is an extension of the Hypertext Transfer Protocol (HTTP). It uses encryption for secure communication ".

Reply:
@majenny pls close this comment thread.  latest https://github.com/MicrosoftDocs/edge-developer/pull/2574/files has:
```
Requests for Compose are sent over a secure HTTPS connection. Each request contains the input prompt, formatting selections, and IDs that are associated with the user's Microsoft Edge profile and device, to reference for subsequent requests.
```